production.env

```diff
@@ -114,10 +105,8 @@ ENABLE_TWOFACTOR=True
 ENABLE_TWOFACTOR_STRICT_ADDRESS=False
 ENABLE_PEERS=True
 MIG_PASSWORD_POLICY="MEDIUM"
-ENABLE_LOGROTATE=False
```

Comment:
I guess you want these to be forwarded as well to avoid log rotation inside containers?

Reply:
nice, thanks for the finding! I added it to production.env